### PR TITLE
Html bolding

### PR DESCRIPTION
**Pull request by danamansana**

### CODE REVIEW

File: src/app/utils/SearchModel.js [hunk 1]
```diff
@@ -192,8 +192,8 @@ const fetchItem = (item, searchRequest) => {
     };
   }
 
-  const modeledTitle = stripPossibleHTMLTag(fetchItemFeature(item, 'title'));
-  const modeledSnippet = stripPossibleHTMLTag(fetchItemFeature(item, 'snippet'));
+  const modeledTitle = stripPossibleHTMLTag(fetchItemFeature(item, 'htmlTitle'));
```
Comment (ktp242) [inline src/app/utils/SearchModel.js]: I am actually surprised that `stripPossibleHTMLTag` didn't strip out `<b>` and `</b>`.

**Comment by ktp242** [comment]
It works fine, but it is what I felt a bit worried as `stripPossibleHTMLTag` not seem to work properly.

The second thing is it returns the `<b>` as the tag. A better way, still, I think is `<strong>` or `<em>`, but maybe we should leave it for now (I guess we should let UX know this?)

**Comment by danamansana** [comment]
I was thinking about replacing the `b` tags with `strong` but I'd like to stay away from parsing under the circumstances. I took a look at the results and visually it doesn't seem any different. But yes, we can talk with the UX team

**Comment by ktp242** [comment]
Actually, the other issue I am not sure if it is an issue, is that the snippets now have a lot of `<br>` cause unnecessary separations of it. Visually the differences are subtle, but you will find the lines now break with `<br>`s.

Compare your local and what on Prod
http://local.nypl.org:3001/search/somali/
https://www.nypl.org/search/somali

You can see the snippet of this `The African Diaspora in the Indian Ocean World` breaks in the middle but not the end of the layout's border.

I am not sure if JAW will pick this up (I don't think so). As we want to keep our HTML as tidy as we can, it is a bit frowned upon.

File: src/app/utils/SearchModel.js (hunk 1 above)
Comment (ktp242) [reply src/app/utils/SearchModel.js]: Actually! I just realized that it actually returns real `<` and `>` but not the decoded one. That's why `stripPossibleHTMLTag ` is not working.

Comment (danamansana) [reply src/app/utils/SearchModel.js]: I think it is using the wrong regular expression, if you hit the API directly you can see the HTML title looks like this example: ```"\u003cb\u003eCoffee\u003c/b\u003e &amp; Connections | The New York Public Library"```

**Comment by danamansana** [comment]
Very interesting, good catch! Personally I think the Google version looks better but that does seem like a UX issue. If it's no good we'll have to be a bit more clever.

**Pull request closed without merging**